src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/tests/stores/ListStore.test.js

```diff
@@ -266,6 +266,98 @@ test('The user store should be called correctly when changing the schema', () =>
     });
 });
 
+test('The userSchema should include schema properties that are not present in the schemaSetting of the user', () => {
+    const loadingStrategy = new LoadingStrategy();
+    const structureStrategy = new StructureStrategy();
+    const page = observable.box(1);
+    const locale = observable.box();
+    const additionalValue = observable.box(5);
```

Comment:
The `locale` and `additionalValue` do nothing in that test, right? Then I would prefer to not have them in here, because I'd like to keep the tests minimal for whatever they are testing.